sync_data/settings.py

```diff
@@ -3,7 +3,19 @@
 DEBUG = False
 
 ACTBLUE_TO_KNACK_MAPPING = {
-    'contribution#createdAt': '638',
-    'lineitems#0#amount': '639',
-    'contribution#contributionForm': '643'
+    'contribution#createdAt': '888',
```

Comment:
settings.py is for prod values - if we're loading stuff into the test db (which is how the application is currently set up), we'll need to add values to test_settings.py